setup.py

```diff
@@ -81,8 +81,8 @@ def get_long_description():
     license="MIT",
     packages=find_packages(where="src", exclude=["*tests*"]),
     install_requires=["typer[all]", "netqasm==0.13.1", "pydantic>=2.1.1", "pydantic-settings",
-                      "tabulate", "jsonschema==4.17.3", "pyyaml", "typing-extensions", "apistar",
-                      "typesystem==0.2.4", "pyjwt"],
+                      "tabulate", "jsonschema>=4.19.0", "referencing", "pyyaml", "typing-extensions",
```

Comment:
validating an experiment (qne experiment validate experiment-name) fails.
I get: Unhandled exception: <WrappedReferencingError Unresolvable(ref='file:asset.json')>
It looks like the reference to asset.json in experiment.json schema doesn't work